packages/backend-for-frontend/src/model/errors.ts

```diff
@@ -438,3 +450,110 @@ export function delegatedEserviceNotExportable(
     title: "Delegated Eservice is not exportable",
   });
 }
+
+export function noVersionInEServiceTemplate(
+  eserviceTemplateId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `No version found in Eservice Template ${eserviceTemplateId}`,
+    code: "noVersionInEServiceTemplate",
+    title: "No version found in Eservice Template",
+  });
+}
+
+export function eserviceTemplateVersionNotFound(
+  eserviceTemplateId: string,
+  eserviceTemplateVersionId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Version ${eserviceTemplateVersionId} not found in Eservice template ${eserviceTemplateId}`,
+    code: "eserviceTemplateVersionNotFound",
+    title: "EService template version not found",
+  });
+}
+
+export function catalogEServiceTemplatePublishedVersionNotFound(
+  eserviceTemplateId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Published version not found in catalog Eservice template ${eserviceTemplateId}`,
+    code: "catalogEServiceTemplatePublishedVersionNotFound",
+    title: "Catalog EService template published version not found",
+  });
+}
+
+export function eserviceTemplateNotFound(
+  eserviceTemplateId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Eservice template ${eserviceTemplateId} not found`,
+    code: "eserviceTemplateNotFound",
+    title: "EService template not found",
+  });
+}
+
+export function eserviceTemplateDataNotFound(
+  eServiceId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Expected template reference in Eservice ${eServiceId} not found`,
+    code: "templateDataNotFound",
+    title: "EService template not found",
+  });
+}
+
+export function eserviceTemplateNotInPublishedState(
+  eserviceTemplateId: EServiceTemplateId
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `EService Template ${eserviceTemplateId} is not in published state`,
+    code: "eserviceTemplateNotInPublishedState",
+    title: "EService Template not in published state",
+  });
+}
+export function eserviceIsNotDraft(eserviceId: string): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `EService ${eserviceId} is not in draft state`,
+    code: "eserviceIsNotDraft",
+    title: "EService is not in draft state",
+  });
+}
+
+export function eserviceTemplateNotPublished(
+  eserviceTemplateId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `EService template ${eserviceTemplateId} is not in published state`,
+    code: "eserviceTemplateIsNotPublished",
+    title: "EService template is not in published state",
+  });
+}
```

Comment:
Duplicated? It seems equal to `eserviceTemplateNotInPublishedState`

Reply:
Thanks! eserviceTemplateNotInPublishedState was not used, removed